### ISSUE TITLE
Options emptyLineBefore can give extra empty line.

### ISSUE THREAD

**Issue by alkorlos** [hudochenkov/stylelint-order#74]
I have:
`stylelint.config.js`
```javascript
module.exports = {
	plugins: 'stylelint-order',
	rules: {
		"order/properties-order": [
			{
				emptyLineBefore: 'always',
				properties: [
					'width',
					'height'
				]
			},
			{
				emptyLineBefore: 'always',
				properties: [
					'font-size',
					'font-family'
				]
			},
			{
				emptyLineBefore: 'always',
				properties: [
					'background-repeat'
				]
			}
		]
	}
};
```

If source css will be
```CSS
.test {
	width: 100%;
	font-size: 14px;
	height: 100%;
	font-family: "Arial", "Helvetica", sans-serif;
	background-repeat: no-repeat;
}
```
Output css will be
```CSS
.test {
	width: 100%;

	height: 100%;

	font-size: 14px;

	font-family: "Arial", "Helvetica", sans-serif;

	background-repeat: no-repeat;
}
```
Too much empty lines, not only between properties groups.

If source css will be
```CSS
.test {
	width: 100%;
	height: 100%;
	font-size: 14px;
	font-family: "Arial", "Helvetica", sans-serif;
	background-repeat: no-repeat;
}
```
Output css will be correct
```CSS
.test {
	width: 100%;
	height: 100%;

	font-size: 14px;
	font-family: "Arial", "Helvetica", sans-serif;

	background-repeat: no-repeat;
}
```

Apparently stylelint-order sets empty line firstly, then sorts properties, doesn’t it? Is it bug?

It can be partially fixed by rule declaration-empty-line-before: never, then at the second lint extra empty line will be deleted, but with warnings.

**Comment by hudochenkov** [comment]
Thank you for reporting.

This is a bug. I don't know why it happens, because first there is a sorting, and then checks for empty lines.

**Comment by hudochenkov** [comment]
Fixed in `stylelint-order@2.0.0`.

**Comment by alkorlos** [comment]
Thank you very much!
